### DISCUSSION TITLE
Zed can setting config open README.md file if there are no open files on project startup

### DISCUSSION THREAD

**Discussion by ihavecoke** [zed-industries/zed#26083] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

I want to ask if the Zed editor has a setting to automatically open the README file by default. This way, when I open a project, the README file will be opened first by default.

**Comment by SomeoneToIgnore** [comment]
Will close in favor of https://github.com/zed-industries/zed/discussions/38228 's description.
